README.md

```diff
@@ -400,6 +400,16 @@ interface Collection {
 }
 ```
 
+If all signatures return/emit/throw/require the same thing, you can consolidate this information in one place:
+
+```js
+interface addEventListener {
+  (type: String, listener: Function | Event, Options?), // assumes Options has been declared before
```

Comment:
Let's simplify this example. The point is not in the details of these various interfaces, but in the shared `=> void` return type, right? Let's clarify that.